pkg/chunk/aws/storage_client.go

```diff
@@ -731,6 +734,7 @@ func (a storageClient) putS3Chunk(ctx context.Context, key string, buf []byte) e
 	})
 }
 
+// Slice of values returned; map key is attribute name
```

Comment:
Thanks for this & similar comments. What do you think about a follow-up PR replacing `string` with typedefs?